incubator/cassandra/templates/cassandra-statefulset.yaml

```diff
@@ -103,6 +103,10 @@ spec:
         volumeMounts:
         - name: data
           mountPath: /var/lib/cassandra
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+        - name: {{ .Values.image.pullSecrets }}
```

Comment:
`image.pullSecrets` should be in the `values.yaml` file, commented out, and documented.